00-demo.md

```diff
@@ -0,0 +1,37 @@
+# Demo Chapter
+
+Here's some code that'll run in a notebook and also run in CI.
+
+```python
+print("Hello world")
+```
+
+## How to Use the Setup
+
+You can write chapters for the course using standard markdown files in a variety of ways. Here are a few approaches that we've found useful. Before getting started, you need to set up a Python environment and install the Python requirements. Once you have Python set up, run `pip install -r requirements.txt` from the root directory to install all the requirements.
```

Comment:
Just to be clear, when yo say "Python environment", do you mean a [Virtual Environment](https://docs.python.org/3/tutorial/venv.html)?